api/proto/teleport/legacy/types/types.proto

```diff
@@ -3665,6 +3665,13 @@ message KubernetesResource {
   string Name = 3 [(gogoproto.jsontag) = "name,omitempty"];
   // Verbs are the allowed Kubernetes verbs for the following resource.
   repeated string Verbs = 4 [(gogoproto.jsontag) = "verbs,omitempty"];
+
+  // APIGroup specifies the API group of the resource. Can be '*' for any,
+  // but doesn't support pattern matching.
+  string APIGroup = 5 [(gogoproto.jsontag) = "api_group,omitempty"];
+
+  // For future use.
```

Comment:
can you write the docs here so we don't have to update them in the future?

Reply:
What did you have in mind for subresource?

Reply:
We need the subresource otherwise we can't prevent access to logs/exec/attach. Currently we are using verbs for that but the native kubernetes way is to use subresources